src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/task/UpdateKpis.kt

```diff
@@ -28,13 +28,13 @@ class UpdateKpis(
     lockAtLeastFor = defaultLockAtLeastFor,
     lockAtMostFor = defaultLockAtMostFor,
   )
-  fun updateKpis() {
+  fun updateKpis(): Unit = runBlocking {
     val day = LocalDate.now()
 
-    val kpiExists = runBlocking { kpiRepository.existsById(day) }
+    val kpiExists = kpiRepository.existsById(day)
     if (kpiExists) {
       LOG.debug("KPI for $day already exists, skipping...")
-      return
+      return@runBlocking
```

Comment:
What's this doing? Looks like a label jump, but what's the difference with a plain `return` in this case?

Reply:
it's just what you have to do apparently; intellij makes you do this. `return` would exit the function, but `return@runBlocking` exits the "block" rather than the whole function

Reply:
I'm not going to go against IntelliJ then 😅 ... as long as the behaviour didn't change and this return skips the INSERT 👍